src/utils/index.js

```diff
@@ -30,13 +30,11 @@ export const capitalize = (str) => {
 export const parseFiltersToURL = (filters) => Object.keys(filters)
     .filter((key) => Array.isArray(filters[key]) ? filters[key].length : !!filters[key]) // Remove falsy values
     .map((key) => {
-        if (filters[key]) {
-            if (Array.isArray(filters[key])) {
-                return filters[key]
-                    .map((val) => `${key}[]=${encodeURIComponent(val)}`)
-                    .join("&");
-            } else return `${key}=${encodeURIComponent(filters[key])}`;
-        } else return "";
+        if (Array.isArray(filters[key])) {
+            return filters[key]
+                .map((val) => `${key}[]=${encodeURIComponent(val)}`)
+                .join("&");
+        } else return `${key}=${encodeURIComponent(filters[key])}`;
```

Comment:
why the change?

Reply:
Coverage, since we moved to Codecov's Github app the coverage workflows would prevent this PR from being merged normally. Besides that, the change is technically a no-op since the check that was removed was always true (because of the previous falsy value filtering). It is not related to this PR but was done in order to get the coverage to pass. I believe I have lowered the threshold but it still does not hurt.